proto/api.proto

```diff
@@ -51,6 +52,49 @@ service Api {
     }
   };
 
+  // Drift V2
```

Comment:
consider moving everything v2 to the bottom of this service class. I think that'll be easier to read compared to having them interpersed

Reply:
good idea, done